src/components/DefinitionEntry.js

```diff
@@ -91,6 +103,26 @@ export default class DefinitionEntry extends React.Component {
     return licenseExpression ? <span>{licenseExpression}</span> : <span>&nbsp;</span>
   }
 
+  getRevisionUrl(definition) {
+    const coordinate = get(definition, 'coordinates')
+    switch (coordinate.provider) {
+      case 'github':
+        return ''
```

Comment:
github has revisions. they are commit hashes and you can get to the state of the repo for a given commit.

Reply:
The code was structured so source url was shown in the the TwoLineEntry. Due to this fact I decided not to have a github revision url because it would be the exact same as the source url. However, I removed the source url from the TwoLineEntry, so github revision now has the proper url template.